ansys/mapdl/reader/rst.py

```diff
@@ -2413,7 +2413,7 @@ def element_solution_data(self, rnum, datatype, sort=True, **kwargs):
 
         return enum, element_data, enode
 
-    def principal_nodal_stress(self, rnum):
+    def principal_nodal_stress(self, rnum, nodes = None):
```

Comment:
```suggestion
    def principal_nodal_stress(self, rnum, nodes=None):
```

Reply:
Delete the whitespace front and after "=" ?